pkg/config/piped.go

```diff
@@ -712,6 +712,10 @@ type PlatformProviderLambdaConfig struct {
 	// If empty, the environment variable "AWS_PROFILE" is used.
 	// "default" is populated if the environment variable is also not set.
 	Profile string `json:"profile,omitempty"`
+	// The interval to fetch the live state of Lambda functions.
+	// Default is 15s.
+	// To reduce AWS API calls, this interval should be larger.
+	LiveStateInterval Duration `json:"liveStateInterval,omitempty" default:"15s"`
```

Comment:
I think `liveStateInterval` will not change while migrating to the plugin.

Plugin will change the name of the LiveState feature??

Reply:
No, I mean if we allow some name like this in the configuration, then we would have something like `driftDetectionInterval` later, which I think should not be here (in the piped.platformProvider config). Livestate or Drift detection config should not be related directly to platform, is what I tried to explain. Name like `awsAPICallInterval` is more relevant.

Reply:
Also, I want to ensure we do not add names/configs that will be changed too much later due to the migrating cost when the user moves to use the plugin-piped. That's why I said, I don't think the used name is good.

Reply:
I almost agree with @khanhtc that we can name this configuration a generic word, but I think awsAPICallInterval is a bit confusing because it sounds like a rate-limiting feature.
We call AWS APIs in other functionality, such as deployment, and they are not affected by this configuration.
What about like awsAPIPollingInterval?

Reply:
The name `awsAPIPollingInterval` is lgtm 👍

Reply:
@khanhtc1202 @Warashi 
Thank you, I changed to `awsAPIPollingInterval`